heed/src/env.rs

```diff
@@ -276,6 +276,53 @@ pub enum CompactionOption {
 }
 
 impl Env {
+    /// Return the size used by all the databases in the environment.
+    pub fn size(&self) -> Result<u64> {
```

Comment:
Could we find a better/more explicit name for this function?
```suggestion
    /// Returns the size used by all the databases on the disk in the environment without taking the free pages into account.
    pub fn real_disk_space_size(&self) -> Result<u64> {
```